src/HUDTeam/DrawManagerImpl.java

```diff
@@ -181,4 +182,63 @@ public static void drawRect(final int x, final int y, final int width, final int
         backBufferGraphics.setColor(color);
         backBufferGraphics.fillRect(x, y, width, height);
     } // by Saeum Jung - TeamHUD
+
+    /**
+     * Draws 2P's lives on screen.
+     *
+     * @param screen
+     *            Screen to draw on.
+     * @param livestwo two
+     *            2P's lives
+     *
+     * by Lee HyunWoo - TeamHUD
+     */
+    public static void drawLives2P(final Screen screen, final int livestwo) {
+        backBufferGraphics.setFont(fontRegular);
+        backBufferGraphics.setColor(Color.WHITE);
+        Entity heart = new Entity(0, 0, 13 * 2, 8 * 2, Color.BLUE) {
+
+        };
+        heart.setSpriteType(SpriteType.Heart);
+
+        int heartWidth = 13 * 2;
+        int startingXPosition = screen.getWidth() - (heartWidth * livestwo) - 20;
+
+        for (int i = 0; i < livestwo; i++)
+            drawEntity(heart, startingXPosition + 30 * i, 10);
+    }
```

Comment:
Currently, the health of the ship on the left is displayed on the right, and the health of the ship on the right is displayed on the left, which makes it difficult to view. Would it be possible to fix this? It would also be great if you could reflect the status as well.

Reply:
> Currently, the health of the ship on the left is displayed on the right, and the health of the ship on the right is displayed on the left, which makes it difficult to view. Would it be possible to fix this? It would also be great if you could reflect the status as well.

I completed the change based on what you suggest me.
Please test it.
Thank you.

Reply:
> Currently, the health of the ship on the left is displayed on the right, and the health of the ship on the right is displayed on the left, which makes it difficult to view. Would it be possible to fix this? It would also be great if you could reflect the status as well.

After editing it, I tested it later
I think it would be good to change the starting position of 1P and 2P in a game for two, so I'll put it back to where it was.